src/pages/landing/weekend-trading/components/_hero.js

```diff
@@ -123,13 +118,7 @@ const HeroComponent = ({ title, content, background_data }) => {
                     <HeroContent direction="column" justify="flex-start">
                         <Header type="subtitle-1">{content}</Header>
                     </HeroContent>
-                    <TryButton
-                        target="_blank"
-                        rel="noopener noreferrer nofollow"
-                        type="submit"
-                        secondary="true"
-                        to={'/signup/'}
-                    >
+                    <TryButton secondary="true" to={'/signup/'}>
```

Comment:
```suggestion
                    <TryButton secondary="true" to="/signup/">
```

Reply:
The usual convention is to write the path in double quotes.

Reply:
done